src/pages/accounts/NewPasswordForm.js

```diff
@@ -23,97 +28,140 @@ function ResetNewPassword() {
+                  />
+                  <div
+                    className={`text-xs py-1/2 text-red-500 ${
+                      errors?.password ? 'opacity-100' : 'opacity-0'
+                    }`}
+                  >
+                    {(errors?.password && errors.password.message) || ''}
+                  </div>
+                </div>
+                <div className="relative w-full mb-2">
+                  <label htmlFor="password2" className="font-bold text-sm">
+                    Confirm password{' '}
+                    <span className="text-red-500 font-bold">*</span>:
+                  </label>
+                  <input
+                    type="password"
+                    id="password2"
+                    ref={pass2}
+                    name="password2"
+                    placeholder="Confirm password"
+                    required
+                    {...register('password2', {
+                      validate: (value) =>
+                        value === password.current ||
+                        'The passwords do not match'
+                    })}
+                    className={`w-full rounded-md placeholder:text-sm placeholder:font-light ${
+                      !errors?.password2 ? '' : 'border-gray-400'
+                    } border-2 focus:outline-none px-2 py-1 font-bold 
               ${!errors?.password2 ? 'border-green-500' : ''} ${
-              errors?.password2 ? 'border-red-500' : ''
-            }
+                      errors?.password2 ? 'border-red-500' : ''
+                    }
               `}
-          />
-          <div
-            className={`text-xs py-1/2 text-red-500 ${
-              errors?.password2 ? 'opacity-100' : 'opacity-0'
-            }`}
-          >
-            {(errors?.password2 && errors.password2.message) || ''}
-          </div>
-        </div>
-        <button className="py-1 px-3 text-center bg-primary hover:bg-hover transition-all hover:transition-all uppercase text-white font-bold rounded-md">
-          {loading ? 'SENDING...' : 'SUBMIT'}
-        </button>
-      </form>
+                  />
+                  <div
+                    className={`text-xs py-1/2 text-red-500 ${
+                      errors?.password2 ? 'opacity-100' : 'opacity-0'
+                    }`}
+                  >
+                    {(errors?.password2 && errors.password2.message) || ''}
+                  </div>
+                </div>
+                <button className="py-1 px-3 text-center bg-primary hover:bg-hover transition-all hover:transition-all uppercase text-white font-bold rounded-md">
+                  {loading ? 'SENDING...' : 'SUBMIT'}
+                </button>
+              </form>
+            </>
+          ) : (
+            <div>
+              {' '}
+              <>
+                <h3 className="text-2xl font-bold text-center py-3 font-rale">
+                  Invalid Token
```

Comment:
I think we can change this message to something else that the user can understand. The user may not understand what the token is.